<launch>
  <!-- 启动规划节点 -->
  <node pkg="astar" type="astar" name="astar_node" output="screen">
    <rosparam command="load" file="$(find astar)/config/astar.yaml" />
  </node>
  
  <!-- 启动 RViz -->
  <!-- 确保 planner.rviz 中 Global Options 的 Fixed Frame 是 "map" -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find astar)/config/planner.rviz" />
</launch>